guides/concepts/transactions.md

```diff
@@ -49,13 +49,27 @@ Between ledger closings, all the nodes in the network are collecting transaction
 
 3. **Submitting**: After signing, the transaction should be valid and can now be submitted to the Stellar network. Transactions are typically submitted using [horizon](https://www.stellar.org/developers/horizon/reference/transactions-create.html), but you can also submit the transaction directly to an instance of [stellar-core](https://github.com/stellar/stellar-core).
 
-4. **Propagating**: Once stellar-core receives a transaction, either given to it by a user or another stellar-core, it does preliminary checks to see if the transaction is valid. Among other checks, it makes sure that the transaction is correctly formed and the source account has enough to cover the transaction fee. Stellar-core doesn't check things that require inspecting the state of the ledger beyond looking up the source account—e.g., that the destination account to which the transaction is trying to send exists, that the account has enough of this asset to sell, that it's a valid path.
-If the preliminary checks pass, then stellar-core propagates the transaction to all the other servers to which it's connected. In this way, a valid transaction is flooded to the whole Stellar network.
+4. **Propagating**: Once stellar-core receives a transaction, either from a user or another
+   stellar-core, it does preliminary checks to see if the transaction is valid by [checking the
```

Comment:
it also checks if the transaction is valid: source account exists, sequence number is good, minimum fee, etc - this got dropped from this change